js/classes/entities/midground/enemies/enemy_base.js

```diff
@@ -55,7 +62,7 @@ class EnemyBase {
      * is within their maximum reaction distance. 
      */
     static get AGGRESSIVE_STANCE() {
-        return "aggressive";
+        return "aggressiveMelee";
```

Comment:
What was the purpose of this change? I ask because depending on what you're trying to do there may be a better way than modifying the stance value.